modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/typeclasses/Environment.kt

```diff
@@ -0,0 +1,9 @@
+package arrow.effects.typeclasses
+
+import arrow.Kind
+
+interface Environment<F> {
+  fun dispatchers(): Dispatchers<F>
+
+  fun handleAsyncError(e: Throwable): Kind<F, Unit>
```

Comment:
What is this needed for?

Reply:
This will be used in `BIO` for async errors that do not belong in the user domain. `cats-bio` logs them. I give the user a choice to do whatever they want with it. (B)IO defaults to log.